lib/acpi/acpi_shell.c

```diff
@@ -104,7 +104,14 @@ static void dump_dev_res(const struct shell *sh, ACPI_RESOURCE *res_lst)
 			shell_print(sh, "ACPI_RESOURCE_TYPE_EXTENDED_ADDRESS64");
 			break;
 		case ACPI_RESOURCE_TYPE_EXTENDED_IRQ:
+			ACPI_RESOURCE_EXTENDED_IRQ * ext_irq_res = &res->Data.ExtendedIrq;
+
 			shell_print(sh, "ACPI_RESOURCE_TYPE_EXTENDED_IRQ");
+			shell_print(sh, "Triggering:%x, Polarity:%x, Shareable:%x,",
+				    ext_irq_res->Triggering, ext_irq_res->Polarity,
+				    ext_irq_res->Shareable);
+			shell_print(sh, "InterruptCount:%d, Interrupts[0]:%x",
+				    ext_irq_res->InterruptCount, ext_irq_res->Interrupts[0]);
```

Comment:
Please use the same style as for other resources, i.e. each line after `ACPI_RESOURCE_TYPE_...` is indented with a tab (`\t`). Also, each variable on its own line, with a space after `:`.